### PR TITLE
fix _expanded to work

### LINKED ISSUE #2057: `AccordionItem` background color is not applied.
### Provide the exact quote of the error or issue

Accordion background color is not applied in the document.

### `Accordion`  running on documentation
image

### `Accordion`  running on Storybook
image

### Browser
Chrome

### Operating System
Windows

This bug may be caused by the browser or the operating system.
I apologize if my perception is incorrect.

### Does the docs page already exist? Please link to it.

https://yamada-ui.com/ja/components/disclosure/accordion#accordionlabel%E3%82%92%E4%BD%BF%E3%81%86

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### PR DESCRIPTION
**Pull request by cidkumagai**
Closes #2057  

## Description
fix _expanded to work

### CODE REVIEW

**Review by 108yen — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
@cidkumagai @108yen 

Hi, thanks for the PR.

You removed `aria-expanded` and now you want to reinstate it?

Shouldn't we use `data-expanded`?

**Comment by cidkumagai** [comment]
@hirotomoyamada 
Yes, I agree. We should use data-expanded. I will make the necessary changes.

Thank you.

File: packages/components/accordion/src/accordion-item.tsx [hunk 1]
```diff
@@ -143,12 +143,13 @@ export const AccordionItem = forwardRef<AccordionItemProps, "li">(
         ref: mergeRefs(register, ref),
         type: "button",
         disabled: isDisabled,
+        "data-expanded": ariaAttr(isOpen),
```
Comment (hirotomoyamada) [inline packages/components/accordion/src/accordion-item.tsx:146]: Change `ariaAttr` to `dataAttr`.

**Comment by cidkumagai** [comment]
@hirotomoyamada 
I have addressed the comments you provided and made the necessary corrections. Could you please review it again?

**Review by setodeve — DISMISSED** [review]

**Review by Haru-0001 — APPROVED** [review]

File: packages/components/accordion/tests/accordion.test.tsx [hunk 2]
```diff
@@ -32,11 +32,61 @@ describe("<Accordion />", () => {
       </Accordion>,
     )
 
+    const button = screen.getByRole("button", { name: /Accordion Label 1/i })
+    expect(button).toHaveAttribute("data-expanded", "")
```
Comment (108yen) [inline packages/components/accordion/tests/accordion.test.tsx:36]: No `""` is needed in other places.
```tsx
expect(button).toHaveAttribute("data-expanded")
```

**Review by 108yen — CHANGES_REQUESTED** [review]

**Review by hirotomoyamada — APPROVED** [review]

**Review by 108yen — APPROVED** [review]

**Pull request merged**